### DISCUSSION TITLE
Option to ignore files in the post processing directory

### DISCUSSION THREAD

**Discussion by reconman** [pymedusa/Medusa#9005] [Ideas & Feature requests]
I have a bunch of manually downloaded files which should be ignored by Post-Processing. I don't want to move them aince I don't want to use 2 different folders for medusa and manual downloads.

Right now I am getting lots of warnings about failed post processing. I have already post processed them myself manually.

I would appreciate an option to be able to mark files in a way where post processing will ignore them.

One option which comes to my mind is a file with a unique name inside of a subfolder, but that only allows ignoring subfolders. Renaming files is not an option, since that would break the torrent seeding.

**Comment by medariox** [comment]
You can set the warning to info, so it won't bother you. Check the settings.

**Reply by reconman** [reply]
Thanks, didn't know about that.
